### ISSUE TITLE
[Bug] `openbb.stocks.options.chains()`: Function does not work from any source.

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#4785]
This function returns an error message and an empty dataframe when `source = Intrinio`. 

Recreate with:

```
openbb.stocks.options.chains('SPY', source = 'Intrinio')
```

and

```
openbb.stocks.options.chains('SPY', source = 'Intrinio', expiration = '2023-04-14')
```

![Screenshot 2023-04-14 at 12 20 14 PM](https://user-images.githubusercontent.com/85772166/232137464-8172a5e4-c67a-46d9-a862-d45302ec4767.png)

No source is working:

![Screenshot 2023-04-14 at 12 29 37 PM](https://user-images.githubusercontent.com/85772166/232138467-37ace9f1-ce51-4aee-831b-0faf17a0a816.png)

**Comment by deeleeramone** [comment]
The functions for getting the full chain in the individual source model files works. Oddly, it appears that Intrinio is missing at least one expiration worth of data when compared to Tradier.

![Screenshot 2023-04-14 at 12 42 47 PM](https://user-images.githubusercontent.com/85772166/232140770-7c3fa527-cfde-420d-b9b1-29d4f984d53a.png)


![Screenshot 2023-04-14 at 12 45 30 PM](https://user-images.githubusercontent.com/85772166/232141121-93572e7b-f9d5-484f-a4ac-5ea9872845d2.png)

**Comment by deeleeramone** [comment]
Operation in `options_sdk_helper.py` is overwriting the dataframe with None.

```
    if (output := None) is None:
        logger.info("Invalid Source or Symbol")
        console.print("Invalid Source or Symbol")
        return 0.0
```

**Comment by deeleeramone** [comment]
related to #4786

**Comment by tehcoderer** [comment]
fixed in #4788
